### PR TITLE
Minor cleanups

### PR DESCRIPTION
**Pull request by BenWeber42**

### CODE REVIEW

File: dusk/grammar.py [hunk 1]
```diff
@@ -510,6 +509,8 @@ def field_index(self, field: DuskField, vindex=None, hindex=None):
         neighbor_iteration = self.ctx.location.current_neighbor_iteration
         field_dimension = self.ctx.location.get_field_dimension(field.sir)
 
+        # TODO: `vindex` is _non-sensical_ if the field is 2d
```
Comment (mroethlin) [inline dusk/grammar.py:512]: correct, but a dawn check to ensure this is about to be merged

File: tests/stencils/test_fields.py [hunk 2]
```diff
@@ -81,3 +82,23 @@ def hv_offsets(
             a = b[Edge, k] + c
             a = b[Edge > Cell > Edge, k + 1] + c[Edge > Cell > Edge, k]
             a = b[Edge, k] + b[Edge, k - 1] + c[Edge > Cell > Edge]
+
+
+@stencil
+def non_sensical_vertical_index_in_2d_field(
```
Comment (mroethlin) [inline tests/stencils/test_fields.py:88]: again, this test will fail soon, c.f. [PR](https://github.com/MeteoSwiss-APN/dawn/pull/1034)

**Review by mroethlin — COMMENTED** [review]
looks good to me. up to you if you want to keep the non-sensical vertically offset 2D fields, they are going to be prohibited from the dawn side soon.

**Comment by BenWeber42** [comment]
I think I'll keep them for now. When I add negative testing, I can just copy & paste them.
Also, I need your explicit approval, otherwise I can't merge. This repo's settings are rather strict, but I want to try to keep the settings like this for now :sweat_smile:

**Review by mroethlin — APPROVED** [review]

**Pull request merged**